cdap-ui/app/cdap/components/Tabs/Tabs.scss

```diff
@@ -41,7 +41,7 @@
       width: 100%;
       display: flex;
       &:first-child {
-        height: fit-content;
+        height: 42px;
```

Comment:
Will this cause issues with other Tab instances?

Reply:
The difference between fit-content and 42px is very small (fit-content was ~42.6px), and we only use horizontal tabs here and in one place in the schema editor. So if the fit-content change was OK for both places, this should be a minor change.

Reply:
@ajainarayanan Want to make sure you see this change to confirm that it is okay for the schema editor!

Reply:
Modified this slightly to still work if scrollbars are always set to on